bin/puppet_profile_generate.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+repo_dir="$(dirname $0)/.."
+. "${repo_dir}/bin/functions"
+app=${1:-undef}
+repo=${2:-https://github.com/example42/pdk-module-template-tp-profile}
+
+show_help () {
+cat << EOF
+
+This script wraps pdk (Puppet Development Kit) to create a new profile for a given application based on Tiny Puppet.
+It requires the pdk command from Puppet Development Kit (https://docs.puppet.com/pdk/)
+You must specify the name of the app for which to create a profile and an optional source git repo.
```

Comment:
I would mention that there is a default set and provide information which url is used as default.